extensions/hibernate-reactive/runtime/src/main/java/io/quarkus/hibernate/reactive/runtime/customized/QuarkusReactiveConnectionPoolInitiator.java

```diff
@@ -36,12 +36,43 @@ public ReactiveConnectionPool initiateService(Map configurationValues, ServiceRe
             // nothing to do, but given the separate hierarchies have to handle this here.
             return null;
         }
-        boolean usePostgresStyleParameters = registry.getService(JdbcEnvironment.class)
```

Comment:
no longer needing `usePostgresStyleParameters` ?

Reply:
ah it was deprected. All good.

Reply:
I guess you've already checked but for the sake of everybody else:
the constructor using that parameter was already deprecated and in Hibernate Reactive we just ignore the value.